freqtrade/rpc/rpc.py

```diff
@@ -1097,19 +1097,28 @@ def _rpc_delete(self, trade_id: int) -> dict[str, str | int]:
                 "cancel_order_count": c_count,
             }
 
-    def _rpc_list_custom_data(self, trade_id: int, key: str | None) -> list[dict[str, Any]]:
-        # Query for trade
-        trade = Trade.get_trades(trade_filter=[Trade.id == trade_id]).first()
```

Comment:
Fgdxn

Reply:
@Bambam-jow please help me understand your comment

Reply:
@arenstar best ignore - it's an account created yesterday - with about a 99% chance of being a scam account.